docs/docs/en/guide/installation/standalone.md

```diff
@@ -4,7 +4,8 @@ Standalone only for quick experience for DolphinScheduler.
 
 If you are a new hand and want to experience DolphinScheduler functions, we recommend you install follow [Standalone deployment](standalone.md). If you want to experience more complete functions and schedule massive tasks, we recommend you install follow [pseudo-cluster deployment](pseudo-cluster.md). If you want to deploy DolphinScheduler in production, we recommend you follow [cluster deployment](cluster.md) or [Kubernetes deployment](kubernetes.md).
 
-> **_Note:_** Standalone only recommends the usage of fewer than 20 workflows, because it uses H2 Database, ZooKeeper Testing Server, too many tasks may cause instability.
+> **_Note:_** Standalone only recommends the usage of fewer than 20 workflows, because it uses in-memory H2 Database in default, ZooKeeper Testing Server, too many tasks may cause instability.
```

Comment:
This is the installation docs. I think we should add it to ```docs/docs/en/development/development-environment-setup.md``` instead of this.

Reply:
Sorry. I've change my mind. I think documentation should be added to both...

Reply:
Sure, I will add this to both. Thx for the suggestion.